src/main/webapp/index.html

```diff
@@ -3,6 +3,55 @@
 
 <title>ZoomTube</title>
 <link rel="stylesheet" href="/style.css" />
-<p>ZoomTube</p>
+<link
+  rel="stylesheet"
+  href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+  integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+  crossorigin="anonymous"
+/>
 
-<a href="/list.html">View list of lectures</a>
+<body>
+  <nav class="navbar navbar-light">
+    <h2 id="header-text">ZoomTube</h2>
+  </nav>
+  <div id="text">
+    <h1 class="display-6">Project Overview</h1>
```

Comment:
Do you think we can replace this with "What is Zoomtube?"

WDYT?

Reply:
I don't have a preference. @amuamushu @n-wach what do you think?

Reply:
sure let's try "What is ZoomTube?"